app/data/hpiRepo.test.ts

```diff
@@ -1,5 +1,5 @@
 // __tests__/hpiRepo.test.ts
-import { hpi2000Repo } from "./hpiRepo"; // Adjust the import according to your file structure
+import { hpi1999Repo } from "./hpiRepo"; // Adjust the import according to your file structure
 import prisma from "./db"; // Your Prisma setup file
```

Comment:
nit: These comments look like leftover AI fragments 🗑️ 🧹

Reply:
Heh I caught them but then thought removing them in this PR might be bad practice, have now cleaned them up 🧹